js/battledata.js

```diff
@@ -856,8 +867,9 @@ var Tools = {
 		}
 		// if there is no entry or enough data in pokedex-mini.js or the animations are disabled or past gen, use the proper sprites
 		gen = (gen === 'xy')? 'bw' : gen;
+		// Do this here in case gen was XY but animations were disabled.
+		Tools.loadSpriteData('bw');
```

Comment:
"In case" scares me. Can't we wrap this in an `if`-statement detecting that case?